src/partials/_faq.hbs

```diff
@@ -0,0 +1,92 @@
+<section class="faq">
+  <h2 class="faq__heading">FAQ</h2>
+
+  <div class="faq__list__links">
+    <a class="faq__link" href="javascript:showonlyone('newboxes1');">Retailers</a>
+    <a class="faq__link" href="javascript:showonlyone('newboxes2');">Parents</a>
+  </div>
+
+  <div class="newboxes" id="newboxes1" style="display: block;padding: 5px;">
+    <div class="list1">
+      <section class="faq__box">
```

Comment:
Regarding your markup choices here, another option would be to use a `<dl>` with `<dt>` as the question and `<dd>` as the answer. Nothing necessarily wrong with using `<div>`s and `<sections>`s. Just something to explore!